plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -732,13 +724,54 @@ function OPDSBrowser:showDownloads(item)
                         G_reader_settings:saveSetting("download_dir", path)
                         UIManager:nextTick(function()
                             UIManager:close(self.download_dialog)
-                            self:createNewDownloadDialog(path, buttons)
+                            self:createNewDownloadDialog(path, filename, buttons)
                             UIManager:show(self.download_dialog)
                         end)
                     end,
                 }:chooseDir()
             end,
         },
+        {
+            text = _("Change file name"),
```

Comment:
Or just `Save as…` like people will be familiar with from other programs?

Reply:
In text editors on `Save as...` we give a name and **save**.
Here we must press a file type button to save (download).

Maybe, `Rename book`.
Or just `Book name` (below is a button `Book information`).

Reply:
The button text is a continuation of the info text "Download file name"...